.github/workflows/ci.yml

```diff
@@ -12,17 +12,18 @@ jobs:
       matrix:
         javaversion: ["8", "9", "10", "11", "12", "13", "14", "15", "16", "17"]
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Set up Java ${{ matrix.javaversion }}
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v2
         with:
+          distribution: "zulu"
```

Comment:
Can we use `temurin` here instead? Seems to match better with what we use internally.